deploy/runtime/defaults/Flat Plate PV_Merchant Plant.txt

```diff
@@ -1100332,19 +1100332,19 @@ total_direct_cost
 1
 1
 1
-5.15027e+12
+42746896
 total_indirect_cost
 1
 1
 1
 1
-6500290
+6.50029e+06
```

Comment:
total_indirect_cost no longer includes sales tax. Total installed is now the sum of total_direct_cost, sales_tax_total, and total_indirect_cost

Reply:
gotcha